### PR TITLE
fix: social login shows `undefined`

### PR DESCRIPTION
**Pull request by magiziz**
# Description

If providing empty `auth.socials` array it'll display "Continue with undefined..." text

## Type of change

- [x] Chore (non-breaking change that addresses non-functional tasks, maintenance, or code quality improvements)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Associated Issues

For Linear issues: Closes APKT-986

# Showcase (Optional)

**Before**

socials-bug

# Checklist

- [x] Code in this PR is covered by automated tests (Unit tests, E2E tests)
- [x] My changes generate no new warnings
- [x] I have reviewed my own code
- [x] I have filled out all required sections

### CODE REVIEW

File: apps/laboratory/tests/shared/utils/project.ts [hunk 1]
```diff
@@ -55,7 +56,8 @@ const ETHERS5_DISABLED_TESTS = [
   'smart-account.spec.ts',
   'wallet-features.spec.ts',
   'siwe-email.spec.ts',
-  'siwe-sa.spec.ts'
+  'siwe-sa.spec.ts',
+  'no-socials.spec.ts'
```
Comment (tomiir) [inline apps/laboratory/tests/shared/utils/project.ts:60]: why disable it?

**Review by zoruka — COMMENTED** [review]
don't we need this tests also for solana?

File: apps/laboratory/src/utils/DataUtil.ts [hunk 2]
```diff
@@ -61,6 +66,11 @@ export const ethersSdkOptions = [
     title: 'Without Email',
     link: '/library/ethers-no-email/',
     description: 'Configuration using ethers without email'
+  },
+  {
+    title: 'Without Socials',
+    link: '/library/ethers-no-socials/',
+    description: 'Configuration using ethers without socials'
```
Comment (enesozturk) [inline apps/laboratory/src/utils/DataUtil.ts:73]: @magiziz Can you add for ethers5 and solana as well as we have email feature for those?

File: apps/laboratory/tests/shared/utils/project.ts (hunk 1 above)
Comment (enesozturk) [reply apps/laboratory/tests/shared/utils/project.ts:60]: + ethers5 should have the feature parity with wagmi/ethers

Comment (magiziz) [reply apps/laboratory/tests/shared/utils/project.ts]: Yeah added test for solana as well 🙏

File: apps/laboratory/src/utils/DataUtil.ts (hunk 2 above)
Comment (magiziz) [reply apps/laboratory/src/utils/DataUtil.ts:73]: Added 🙏

**Review by zoruka — APPROVED** [review]

**Review by tomiir — APPROVED** [review]

**Review by enesozturk — APPROVED** [review]

**Pull request merged**